components/member_list_team/member_list_team.tsx

```diff
@@ -58,14 +59,11 @@ export default class MemberListTeam extends React.Component<Props, State> {
         };
     }
 
-    componentDidMount() {
-        this.props.actions.loadProfilesAndTeamMembers(0, Constants.PROFILE_CHUNK_SIZE, this.props.currentTeamId).then(({data}) => {
-            if (data) {
-                this.loadComplete();
-            }
-        });
-
+    async componentDidMount() {
+        await this.props.actions.loadProfilesAndTeamMembers(0, Constants.PROFILE_CHUNK_SIZE, this.props.currentTeamId);
```

Comment:
Do these need to be done one at a time? Perhaps this could be wrapped with `Promise.all` to do them both at once like
```javascript
await Promise.all([
    doSomething(),
    doSomethingElse(),
]);
```

Reply:
Thank you! Great catch.